scripts/nesi_scripts/vm_params_gen.sl

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# script version: slurm
+#
+# must be run with sbatch vm_gen.sl [realisationDirectory] [OutputDirectory] [managementDBLocation]
+
+#SBATCH --job-name=VM_GEN
+#SBATCH --time=00:15:00
+#SBATCH --cpus-per-task=1
+
+if [[ -n ${CUR_ENV} && ${CUR_HPC} != "mahuika" ]]; then
+    source $CUR_ENV/workflow/install_workflow/helper_functions/activate_env.sh $CUR_ENV "mahuika"
+fi
+
+REL_CSV=$1
+OUT_DIR=$2
+VM_VERSION=$3
+VM_TOPO=$4
+HH=$5
+REL_NAME=$6
+MGMT_DB_LOC=$7
+
+
+
+if [[ ! -d $OUT_DIR ]]; then
+    mkdir -p $OUT_DIR
+fi
+
+#updating the stats in managementDB
+if [[ ! -d $MGMT_DB_LOC/mgmt_db_queue ]]; then
+    #create the queue folder if not exist
+    mkdir $MGMT_DB_LOC/mgmt_db_queue
+fi
+timestamp=`date +%Y%m%d_%H%M%S`
+python $gmsim/workflow/scripts/cybershake/add_to_mgmt_queue.py $MGMT_DB_LOC/mgmt_db_queue $REL_NAME VM_PARAMS running $SLURM_JOB_ID
+
+runtime_fmt="%Y-%m-%d_%H:%M:%S"
+start_time=`date +$runtime_fmt`
+echo $start_time
+
+echo "python $gmsim/Pre-processing/VM/rel2vm_params.py --hh $HH --vm-version $VM_VERSION --vm-topo  $VM_TOPO --pgv 2.0 --ds-multiplier 0.75 $REL_CSV"
+python $gmsim/Pre-processing/VM/rel2vm_params.py --hh $HH --vm-version $VM_VERSION --vm-topo  $VM_TOPO --pgv 2.0 --ds-multiplier 0.75 $REL_CSV
```

Comment:
Having pgv 2.0/ds mult 0.75 is ok for now, but does need to be able to take the validation parameters later

Reply:
Agreed. They should be part of the VM parameter section.